### PR TITLE
Converting SalesforceHybrid tests to new testing framework

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">📱 Needs testing on a real device if change is non-trivial.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_PR-Check">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/SmartStoreLoadJSTest.java [hunk 1]
```diff
@@ -26,26 +26,37 @@
...
 
+import java.util.Arrays;
+import java.util.List;
 
 /**
- * Running javascript tests for SmartStore plugin
+ * Running javascript tests for SmartStore plugin.
  */
+@RunWith(AndroidJUnit4.class)
+@MediumTest
```
Comment (wmathurin) [inline libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/SmartStoreLoadJSTest.java]: Shouldn’t it be large?

Comment (bhariharan) [reply libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/SmartStoreLoadJSTest.java]: Not sure if it's based on runtime of the suite or number of tests in the suite. I can change it to large.

Comment (wmathurin) [reply libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/SmartStoreLoadJSTest.java]: Number of tests is something the runner can easily figure out. It has to do with the runtime characteristics of the tests. Here is an old blog post about it: https://testing.googleblog.com/2010/12/test-sizes.html?m=1

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | Classname | Time|
--- | --- | ---|
No tests found. | com.android.builder.testing.ConnectedDevice | 0.001|


# Tests results for SalesforceHybrid

<p align="right" data-meta="generated_by_SalesforceHybrid">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Tests did not run to completion.</td>
    </tr>
  </tbody>
</table>

# Tests results for SmartStore

<p align="right" data-meta="generated_by_SmartStore">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by bhariharan** [comment]
All tests are passing now.

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | Classname | Time|
--- | --- | ---|
No tests found. | com.android.builder.testing.ConnectedDevice | 0.001|


# Tests results for SmartSync

<p align="right" data-meta="generated_by_SmartSync">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**